Directory.Build.props

```diff
@@ -0,0 +1,55 @@
+<Project>

```

Comment:
Our release builds are run on "Package ES" machinery, which in part works by blowing away Directory.Build.props; will the build still function properly if none of these properties are set?

Reply:
Yep, all this should be optional and only kick in when `/p:MsBuildCacheEnabled=true`, which shouldn't be the case in your release builds.

From speaking offline, added a comment to clarify this. Alternately we could move this to custom.props if desired and have this file simply import that file. I'm open to suggestion.